src/components/ProductCardRow/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './ProductCardRow';
```

Comment:
Maybe a better name for this component could be "ProductListSection" or something referencing what is showing in a better way. If I read "row" I usually think of a table structure whereas this is a home page section.
Just a suggestion, not a must at all.

Reply:
I'll change it if you think it's more idiomatic sure. When I think of a list I think of something that goes from the top to the bottom, not something that goes from the left to the right, but I'm not used to frontend naming schemes